### DISCUSSION TITLE
Help window aligned left when nvim-tree is set to right side

### DISCUSSION THREAD

**Discussion by oaflindberg** [nvim-tree/nvim-tree.lua#3026] [Q&A]
I've set my nvim-tree to be on the right side, which works fine. But when opening the help page (g?), that shows on the left side. Is it possible to have that aligned to the right as well?

**Comment by alex-courtis** [comment]
It's hardcoded to row 1 col 0 `NW` anchor.

We could change that to `NE`.

As usual, pull requests are most gratefully appreciated, see [CONTRIBUTING.md](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md)
